### PR TITLE
nrows and chunking

### PR DESCRIPTION
**Pull request by kjmerf**
Hi guys,

This PR should be merged after the circle-ci one. The purpose of this PR is to be able to handle files that pandas cannot read into memory all at once, because they are too big. Issue #24 is logged to track the work.

Summary of updates:

- added nrows option to command line. If you just want see the first 100 rows try: `scroll --nrows=100 titanic.csv`

- added chunksize to command line. If you want to chunk a big file and see 100 rows at a time try: `scroll --chunksize=100 titanic.csv`. To see the next chunk, just press "n"

These are of course trivial examples. To test with a big file, try downloading http://sdm.lbl.gov/fastbit/data/star2002-full.csv.gz.

Limitations:

- Have not added "goto next chunk" to the help screen. Might need @johncmerfeld to assist with that

- When the last chunk is reached, we just stay there. Wondering if you guys think we should go back to the top?

From a coding POV, I added a reader attribute to the DFWindow class and an optional reader argument to the key_press_and_print_df function. Have a look and let me know what you think!

### CODE REVIEW

**Comment by johncmerfeld** [comment]
Haven't checked out the full chunking capability yet, but added its key command to the help box. Also removed the constant values from scroller.py so now, whenever we modify the help_screen.py file, the box will size itself according to how many items there are on the help menu.

**Comment by kjmerf** [comment]
@johncmerfeld good commit but shouldn't next chunk be listed before exit scroller? I switched it, all good.

**Comment by kjmerf** [comment]
@johncmerfeld @baogorek this one is ready for approval

File: datascroller/scroller.py [hunk 1]
```diff
@@ -213,6 +214,14 @@ def query(self, query_string, viewing_area):
         #    table_name = query_words[from_index + 1]
         #    exec("%s = %d" % (table_name, self.full_df))
 
+    def get_next_chunk(self):
+        try:
+            df = self.reader.get_chunk()
+        # if we reach the last chunk, just stop there
+        except StopIteration:
```
Comment (johncmerfeld) [inline datascroller/scroller.py:221]: So, the idea is, if we hit `StopIteration`, then `full_df` is equal to the last chunk. So this function returns a new copy of the same `DFWindow`? Couldn't we just return `self`? 

More importantly, personally I think the default behavior should be to go back to the firs chunk. Probably from a UI perspective it'd be smart to print a "Reached last chunk. Returning to top" message.

**Review by johncmerfeld — APPROVED** [review]
I like the design a lot. Very elegant way to integrate the chunking into the existing class structure. 

Like I said, I think it should loop around when it reaches the end, but we can do that in a later iteration. Great job!

File: datascroller/scroller.py (hunk 1 above)
Comment (kjmerf) [reply datascroller/scroller.py:221]: Makes sense. Let’s consider this one MVP and update behavior in subsequent PR.

**Pull request merged**